uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -531,4 +531,47 @@ public void confirmCallingAppNotVerified() {
             UiAutomatorUtils.handleButtonClick("android:id/button1");
         }
     }
+
+    /**
+     * Check if the Device Code Flow option shows up in sign in flow.
+     * @param tenantId tenant ID to use in Join Tenant
+     * @throws UiObjectNotFoundException
+     */
+    public void checkForDcfOption(@Nullable final String tenantId){
+        final String tenantIdToUse;
+
+        // If no tenant ID is specified, default to microsoft tenant
+        if (tenantId == null) {
+            tenantIdToUse = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
Is this something specific to how this feature works?

Reply:
Just has to be a valid tenant. Basically, we needed a way to call the /authorize end point without passing in loginhint as passing it in will not cause the DCF option to appear. We always pass in login hint in BrokerHost, so to test this, we created a new button that passes in the tenant id but not a loginhint. This allows us to test the flow.
Tagging @mohitc1 to keep me honest since he understands the flow better than I do.

Reply:
What tenant we use doesn't really matter from what I understand